app/assets/stylesheets/building_blocks.scss

```diff
@@ -0,0 +1,83 @@
+html,
+body {
+  background: #FAFAFA;
```

Comment:
Color literals like `#FAFAFA` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FAFAFA` should be written as `#fafafa`